python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -36,10 +36,9 @@
 from lsst.ip.diffim import ImagePsfMatchTask, DipoleAnalysis, \
     SourceFlagChecker, KernelCandidateF, cast_KernelCandidateF, makeKernelBasisList, \
     KernelCandidateQa, DiaCatalogSourceSelectorTask, DiaCatalogSourceSelectorConfig, \
-    GetCoaddAsTemplateTask, GetCalexpAsTemplateTask, DipoleFitTask
+    GetCoaddAsTemplateTask, GetCalexpAsTemplateTask, DipoleFitTask, ImageDecorrelationTask
```

Comment:
Sorry, I only just noticed that you called this, "ImageDecorrelationTask".  This name is misleading, and should be changed.  It decorrelates the effects of an A&L kernel, but doesn't deal with general correlations (e.g. due to warping).